test/views/SettlorIndividualOrBusinessViewSpec.scala

```diff
@@ -17,13 +17,13 @@
 package views
 
 import forms.SettlorIndividualOrBusinessFormProvider
-import models.{NormalMode, IndividualOrBusiness}
+import models.{IndividualOrBusiness, NormalMode}
 import play.api.data.Form
 import play.twirl.api.HtmlFormat
-import views.behaviours.ViewBehaviours
+import views.behaviours.{OptionsViewBehaviours, ViewBehaviours}
 import views.html.SettlorIndividualOrBusinessView
 
-class SettlorIndividualOrBusinessViewSpec extends ViewBehaviours {
+class SettlorIndividualOrBusinessViewSpec extends OptionsViewBehaviours {
```

Comment:
Is this changing the existing implementation for the DeceasedSettlorIndividual Or Business question?

Reply:
This is a new implementation and not part of the deceased settlor journey